### PR TITLE
doc(texts): faq, known issue about editors

### PR DESCRIPTION
**Pull request by ameli90**
Related PR https://github.com/JOOlanders/TextareaPro/pull/7

### CODE REVIEW

File: 01.elements/texts/03.troubleshooting/02.known-issues/docs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Known Issues
+taxonomy:
+    category: docs
+---
+
+* It is not possible to use two different editors, for example JCE and TinyMCE, on the same page for Textarea Pro elements as they conflicts with each other.
```
Comment (miljan-aleksic) [inline 01.elements/texts/03.troubleshooting/02.known-issues/docs.md]: ..."TextareaPro elements as they conflict with each other."

File: 01.elements/texts/01.basics/06.faq/docs.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: F.A.Q.
+taxonomy:
+    category: docs
+---
+
+* **Why editor is not loaded for Textarea Pro element?** <br /> Check the specified editor in the elements settings, ensure that the submission is in trusted mode and the user is logged in.
```
Comment (miljan-aleksic) [inline 01.elements/texts/01.basics/06.faq/docs.md]: "**Why the editor is not loading?**"

Comment (ameli90) [inline 01.elements/texts/01.basics/06.faq/docs.md]: fixed

File: 01.elements/texts/01.basics/06.faq/docs.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: F.A.Q.
+taxonomy:
+    category: docs
+---
+
+* **Why editor is not loading for Textarea Pro element?** <br /> Check the specified editor in the elements settings, ensure that the submission is in trusted mode and the user is logged in.
```
Comment (miljan-aleksic) [inline 01.elements/texts/01.basics/06.faq/docs.md]: Set the question as **Why TextareaPro editor is not loading?**

Comment (ameli90) [inline 01.elements/texts/01.basics/06.faq/docs.md]: done

**Pull request merged**